src/site/_collections/authors.js

```diff
@@ -130,17 +128,10 @@ module.exports = (collections) => {
       invalidAuthors.push(key);
     }
 
-    let authorsImage = path.join('/images', 'authors', `${key}@2x.jpg`);
-    if (process.env.ELEVENTY_ENV === 'prod') {
-      const authorsImageExists = fs.existsSync(path.join('src', authorsImage));
-      if (!authorsImageExists) {
-        console.warn(
-          `No 2x image was found for ${author.title} (${author.key}), replacing with placeholder.`,
-        );
-        authorsImage = path.join('/images', 'authors', 'no-photo.svg');
-      }
+    if (!author.image) {
+      author.image = 'image/admin/1v5F1SOBl46ZghbHQMle.svg';
```

Comment:
Can we put it in a const, like PLACEHOLDER_IMG or so?

Reply:
sure, ill do that now.